how-to-guides/re-run-failed-training/scripts/re_run_failed_training.py

```diff
@@ -0,0 +1,113 @@
+import neptune.new as neptune
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import os
+from torchvision import datasets, transforms
+
+# Step 1: Get Run ID
+
+# Fetch project
+project = neptune.get_project(name='common/showroom', api_token='ANONYMOUS')
+
+# Fetch only inactive runs
+runs_table_df = project.fetch_runs_table(state="idle", tag=['showcase-run']).to_pandas()
+
+# Sort runs by failed 
+runs_table_df = runs_table_df.sort_values(by='sys/failed', ascending=True)
```

Comment:
Seems that this sorting is not necessary. You sort by True/False str, while you want to extract latest run that failed. This sorting does not guarantee that.

Reply:
True!

Removed it